Automattic/jetpack
# Slideshow Block: Change pagination arrow icons and colors

Fixes #27710.

#### Changes proposed in this Pull Request:

* Update the arrow icons to match the gallery carousel icons
* Update the colors to match the gallery carousel colors

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Go to the block editor and insert a new Slideshow
* Save the post and check how the post is rendered on the frontend
* Make sure the style matches the expected output from the "After" column:

| Before | After |
| --- | --- |
| <img width="600" alt="Screen Shot 2022-12-13 at 18 20 31" src="https://user-images.githubusercontent.com/6760046/207414304-04efc935-8dc0-4ed8-b8b6-fc12af0756ea.png"> | Light theme example: <img width="600" alt="Screen Shot 2022-12-14 at 16 37 51" src="https://user-images.githubusercontent.com/6760046/207654496-96b05c29-a42a-4ef6-b809-fd1c34dcb244.png"> |
| | Dark theme example: <img width="600" alt="Screen Shot 2022-12-14 at 16 33 34" src="https://user-images.githubusercontent.com/6760046/207653660-29d7ed17-acb9-4365-9dd3-4767b88caa45.png"> |
